pkg/scheduler/objects/node.go

```diff
@@ -412,7 +417,7 @@ func (sn *Node) preConditions(ask *AllocationAsk, allocate bool) bool {
 func (sn *Node) preAllocateCheck(res *resources.Resource, resKey string) bool {
 	// cannot allocate zero or negative resource
 	if !resources.StrictlyGreaterThanZero(res) {
-		log.Log(log.SchedNode).Debug("pre alloc check: requested resource is zero",
+		getRateLimitedNodeLog().Debug("pre alloc check: requested resource is zero",
```

Comment:
Note: it's not in the PR (yet), but we might do something with L#425-430:
```
	if sn.IsReserved() {
		if !sn.isReservedForApp(resKey) {
			log.Log(log.SchedNode).Debug("pre alloc check: node reserved for different app or ask",
				zap.String("nodeID", sn.NodeID),
				zap.String("resKey", resKey))
			return false
		}
	}
```
Not sure how much output this generates. We can just log the successful reservation (which is NOT logged at all right now, see `Node.Reserve()`) and ignore this.